packages/dara-core/js/utils.ts

```diff
@@ -1,5 +1,14 @@
+/**
+ * Prepend window.dara.base_url to asset path if defined.
+ * This is required for static assets to be loaded correctly when the app is not hosted at the root of the domain.
+ *
+ * Since all static assets are server from `/static/`, this is only applied to asset paths that start with `/static/`.
+ * This way we're not affecting any other assets that are loaded from other domains.
+ *
+ * @param asset asset URL
+ */
 export const prependBaseUrl = (asset: string): string => {
-    if (window.dara?.base_url) {
+    if (window.dara?.base_url && asset.startsWith('/static/')) {
```

Comment:
If they configure static to be set as something else in config.static_files_dir this still results in being in static folder in here so that is fine, right?

Reply:
Yes, so `config.static_files_dir` changes which folder is mounted (default `dist`), but our backend router always mounts it at `/static` URL.